client/src/rubin/nublado/client/models/_extension.py

```diff
@@ -0,0 +1,42 @@
+"""Models for notebook execution via the ``/user/:username/rubin/execute``
```

Comment:
I'm guessing this is probably correct and I'm missing some context, but are these two different endpoints:
```
/rubin/execution
/rubin/execute
```

Reply:
Should be `/execution`